### PR TITLE
Sample data after workflow

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes

The newflash cards now have first lines in font-weight bold. 

After merging the sample data workflow, all articles werde unpublished. This looks awkward. 
A first attempt was a new article with a short description of the workflow as new feature.
But this requires some know how from users and was no good solution.

Now the blog articles are stored as as published and are set featured, so that they are visible at once, as before the workflow data were added. 

If the site is multilingual, there are several workflows. The workflows have the language tag as suffix.
As in sample data multilang, autor-alias is now set to "Joomla".

### CODE REVIEW

File: administrator/language/en-GB/plg_sampledata_blog.ini [hunk 1]
```diff
@@ -24,15 +24,18 @@ PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_4_TITLE="Your Modules"
...
-PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_8_INTROTEXT="<p>We proudly present Joomla Version 4!</p>"
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_8_INTROTEXT="<p><strong>We proudly present Joomla Version 4!</strong></p></p>Learn more about workflows in Joomla.</p>"
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_8_TITLE="Joomla"
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_FULLTEXT=""
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_INTROTEXT="<p>Don't panic - your blog entries are not yet published.<br>We have activated the Workflow for the Sample Data Blog.<br>Workflows help you managing the way of your articles have to go until they are published.<br>You first must publish your articles! Go into your Backend and let go your articles through the different stages of your workflow until they are published. </p>"
```
Comment (richard67) [inline administrator/language/en-GB/plg_sampledata_blog.ini]:
```suggestion
PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_9_INTROTEXT="<p>Don't panic - your blog entries haven't been published yet.<br>We have activated a workflow for the sample data blog.<br>Workflows help you managing the way of your articles have to go until they are published.<br>You first must publish your articles! Go into your backend and let go your articles through the different stages of your workflow until they are published.</p>"
```

Comment (brianteeman) [reply administrator/language/en-GB/plg_sampledata_blog.ini]: >Workflows help you managing the way of your articles have to go until they are published.

==>

Workflows manages the stages your articles must go through until they are published

> Go into your backend and let go your articles through the different stages 

===> 
In the administrator process your articles through the different stages

**Comment by chmst** [comment]
Wasn't me, it is there since years :) but fixed.

**Comment by brianteeman** [comment]
Yeah I saw.

**Comment by richard67** [comment]
@chmst Is it by purpose that all articles which belong to the "Blog" category and handled with the example workflow are featured, except of the "Your Modules" article, which is not featured? Or should the "Your Modules" article be featured, too?

**Comment by chmst** [comment]
It was intended. The base.sql defines the featured blog with 1 leading and 3 intro articles. So at the moment it looks awkward if 
there is one article more. 
I can either change base.sql or update the home menu item everywhere or remove an article or add more than one article, so that the link list makes looks better.

**Comment by richard67** [comment]
> I can either change base.sql or update the home menu item everywhere or remove an article or add more than one article, so that the link list makes looks better.

What I don't understand is the workflow thing: As far as I could see, the final "Published" stage always includes =yes and featured=yes, and this one article then has this final stage but not featured. I see no way to get to that combination of  published=yes and featured=no with the workflow.

Is that a problem? If not, forget my silly question. But if it can be a problem it should be fixed somehow.

**Comment by chmst** [comment]
Thank, very attentive. It was at least ugly. So I have the workflow information now in the category "help" and it is visible in the sub-menu articles. This is not cool at all, but is independent from our cassiopeia.

**Review by richard67 — APPROVED** [review]

**Comment by richard67** [comment]
I have tested this PR :white_check_mark: with success.

Tested both mono-lingual and multi-lingual scenarios.

**Comment by richard67** [comment]
Thanks!

**Pull request merged**